owncloudApp/src/main/res/values/strings.xml

```diff
@@ -755,4 +756,5 @@
     <string name="ocis_accounts_warning_checkbox_message">Don\'t show again</string>
     <string name="ocis_accounts_warning_button">Understood</string>
 
+    <string name="file_already_exists_description">File with name %1$s, already exists.</string>
```

Comment:
Rather than adding new strings at the end of the file, try to place them next to other related strings so that they are all grouped together. In this case, this string could be around line 383, related to the other one you used for the dialog